src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
The fields of an enum class are constant and must be in `UPPER_CASE_SNAKE_CASE` (aka screaming snake case).
```suggestion
    BLACK,
    WHITE,
    RED,
    BLUE
```